PWGEM/PhotonMeson/TableProducer/skimmerGammaConversions.cxx

```diff
@@ -84,6 +107,46 @@ struct skimmerGammaConversions {
     for (auto& lPairIt : fV0ConfirmationLabel) {
       lXaxis->SetBinLabel(lPairIt.first + 1, lPairIt.second.data());
     }
+
+    //This is added in order to access the ccdb
+
+    ccdb->setURL("http://alice-ccdb.cern.ch");
+    ccdb->setCaching(true);
+
+    //apperantly works without all of this :)
+
+    //ccdb->setLocalObjectValidityChecking();                               //no idea wether this is usefull or not
+    //ccdb->setCreatedNotAfter(nolaterthan.value);                          //maybe I decide that this is usefull afterall
+
+    //probably don't need that right here
+    /*
+    auto lut = o2::base::MatLayerCylSet::rectifyPtrFromFile(ccdb->get<o2::base::MatLayerCylSet>("GLO/Param/MatLUT")); //this somehow sets the geometry/calibration file that is read in???
+
+    if (!o2::base::GeometryManager::isGeometryLoaded())
+    {
+      ccdb->get<TGeoManager>("GLO/Config/GeometryAligned");
+      constexpr long run3grp_timestamp = (1619781650000 + 1619781529000) / 2;
+
+      o2::parameters::GRPObject* grpo = ccdb->getForTimeStamp<o2::parameters::GRPObject>("GLO/GRP/GRP", run3grp_timestamp);
+      o2::base::Propagator::initFieldFromGRP(grpo);
+    }
+    else
+    {
+      LOGF(warning, "Geometry not loaded");
+    }
+    */
```

Comment:
I would like to leave it like that if it is ok for you for now and remove it in a later commit, since I first want to see if the CCDB access works like that on the Hyperloop and if it works for different datasets

Reply:
ok :)